cfgov/jinja2/v1/document-detail/index.html

```diff
@@ -43,6 +43,7 @@
 {% set lipsumText = lipsum(1, false, 10, 25) | safe %}
 
 {% block content_main %}
+    {% do global_dict.update({ 'full_width_sans': true }) %}
```

Comment:
had to add full_width_sans to the global dict as wagtail rendered templates were not inheriting the template context variables.